README.md

```diff
@@ -102,16 +89,30 @@ do
 end
 ```
 
-### Mission Setup
+Instead of copying the files to their respective destination as done in the normal [Installation](#installation), it is recommended to create symbolic links for development instead.
+
+This can be done using powershell. Before running the commands, update the paths accordingly.
 
-Add the following script to your mission (adjust the paths to match your repo location):
+Build to make sure all files exist:
 
-```lua
-package.cpath = package.cpath..[[C:\Development\DCS-gRPC\rust-server\target\debug\?.dll;]]
-GRPC = { basePath = [[C:\Development\DCS-gRPC\rust-server\lua\]] }
-dofile(GRPC.basePath .. [[grpc.lua]])
+```bash
+make build
+```
+
+Create directories and links:
+
+```ps1
+New-Item -ItemType SymbolicLink -Path "M:\Saved Games\DCS.openbeta\Scripts\DCS-gRPC" -Value "M:\Development\DCS-gRPC\rust-server\lua"
+New-Item -ItemType SymbolicLink -Path "M:\Saved Games\DCS.openbeta\Scripts\Hooks\DCS-gRPC.lua" -Value "M:\Development\DCS-gRPC\rust-server\lua\grpc-hook.lua"
+New-Item -Path "M:\Saved Games\DCS.openbeta\Mods\Tech\DCS-gRPC" -ItemType "directory"
+New-Item -ItemType SymbolicLink -Path "M:\Saved Games\DCS.openbeta\Mods\Tech\DCS-gRPC\dcs_grpc_server.dll" -Value "M:\Development\DCS-gRPC\rust-server\target\debug\dcs_grpc_server.dll"
+New-Item -ItemType SymbolicLink -Path "M:\Saved Games\DCS.openbeta\Mods\Tech\DCS-gRPC\dcs_grpc_server_hot_reload.dll" -Value "M:\Development\DCS-gRPC\rust-server\target\debug\dcs_grpc_server_hot_reload.dll"
```

Comment:
Todo:
- [ ] use paths that are not specific to my `M:` windows installation

Wdyt about the symbolic links instead of the difference Lua mission code?

Reply:
I would update the README to just include both options and let people choose. Personally I would not use the Symbolic links since I want some missions to use my "dev" version and some missions to use the latest "Stable" version from main so I edit the lua in each of the missions to point to different locations..

Reply:
Kept both 👍